.gitignore

```diff
@@ -179,3 +179,8 @@ CLAUDE.local.md
 #### main branch only stuff below this line, things to backport go above. ####
 # main branch only: ABI files are not checked/maintained.
 Doc/data/python*.abi
+
+# Mypy symlinks (generated by Misc/mypy/make_symlinks.py)
+Misc/mypy/_colorize.py
+Misc/mypy/_pyrepl
+Misc/mypy/token.py
```

Comment:
This doesn't look related to this PR. Could you revert this, and the removal of the files themselves?

Reply:
Hm, I guess working with symlinks is cumbersome, especially if you're on a system where Git replaces them with regular files. (cc @ambv -- FYI)

I've asked Git to restore them using:
```
git restore -WS -s origin/main Misc/mypy/_colorize.py Misc/mypy/_pyrepl Misc/mypy/token.py .gitignore
```
and pushed the result to your PR.